spectrum/test_article.py

```diff
@@ -170,10 +170,9 @@ def _single_relation(from_id, to_id):
         assert len(result['items']) >= 1
         assert result['items'][0]['id'] == recommended.id()
         # load the article page, this will call recommendations
-        article_from_api = checks.API.wait_article(id=article.id())
```

Comment:
Not needed anymore because of `checks.API.wait_recommendations()`?

Reply:
:+1: